react-projects/src/codeImprovement.js

```diff
@@ -0,0 +1,52 @@
+//Exercise 1:
+const average=(numbers)=>numbers.reduce((acc,current)=>acc+current,0)/numbers.length
+console.log(average([1, 2]));
```

Comment:
This line of console.log is not needed. for having a quality codes , you may remove it on this exercise and on the next 8 exercises.